plugins/modules/azure_rm_appgateway_info.py

```diff
@@ -192,9 +212,22 @@ def format_response(self, appgw):
             "location": d.get("location"),
             "operational_state": d.get("operational_state"),
             "provisioning_state": d.get("provisioning_state"),
+            "ssl_policy": None if d.get("ssl_policy") is None else {
+                "policy_type": _camel_to_snake(d.get("ssl_policy").get("policy_type", None)),
+                "policy_name": self.ssl_policy_name(d.get("ssl_policy").get("policy_name", None)),
```

Comment:
"ssl_policy" returns the following information without policy_name. Thank you very much!
```e,g,
'ssl_policy': {'policy_type': 'Custom', 'cipher_suites': ['TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256', 'TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256', 'TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384'], 'min_protocol_version': 'TLSv1_1'}